Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-3: This article shall be inoperative unless it shall have been ratified as an amendment to the Constitution by the legislatures of the several States, as provided in the Constitution, within seven years from the date of the submission hereof to the States by the Congress.   ratified #18
+2: No new State shall be admitted to the Union, which has not established a written Constitution which shall serve as supreme Law in the State, subject not to any law passed by the Congress of these United States, but only to the Constitution of these United States, in which a Republican form of government shall be described; and which shall provide for the election of State Legislature, Representatives to the Congress of these United States, Electors for the Presidency of these United States, and a Chief Executive of the State, by the people of the State, or by their chosen representatives; and which shall provide for the determination of the Citizens of the State to remove themselves from the Union; and which shall recognize the Rights and Responsibilities of the People inherent in their being as are described in Article V; and which shall enshrine in law the duty of the State to protect these Rights, and not to infringe upon them except as punishment of an individual for crimes of which they have been convicted, by unanimous decision of a Jury of disinterested peers, in a court of Law; and which recognizes the State as the sole organization given the sanction to act in aggression in its territory, and then only in accordance with such Constitution and Laws as are approved by the People or their elected Representatives.
 
-Article [XIX] (Amendment 19 - Women's Suffrage Rights)
-The right of citizens of the United States to vote shall not be denied or abridged by the United States or by any State on account of sex. affects 15
+Article V (Article 5 - The Enumeration of the Inherent Rights and Responsibilities of the People)
+The following are recognized as the Rights of the People, which are not granted by any earthly Authority, but inhere in their being as conscious, sentient lives, and which carry with them Responsibilities which cannot be enforced by any earthly Authority.
 
-Congress shall have power to enforce this article by appropriate legislation.   ratified #19
+1: It is the Right of all People to be secure in their person and property from aggression, coercion, unwarranted search and seizure, and theft; and to defend their selves and their property from such aggression, coercion, unwarranted search or seizure, or theft; and to possess, and to acquire by peaceful means, arms and other tools to accomplish that defense; and to dispose of their person and property in any manner they wish; and the Responsibility of all People to avoid, to the best of their ability, causing damage to another Person or that Person's Property without that Person's assent, and to exercise reasonable restraint in defense of their own Person or Property.
```

Comment:
This most important right is a recognition born from English Common Law: That every man's home is his castle. When all other methods of cooperation fail, the ultimate guard against violence is the recognition that two men with an irreconcilable disagreement can both retreat to their own property, and act as they may thereon.